cdi/src/test/java/org/infinispan/cdi/test/event/CacheEventTest.java

```diff
@@ -155,7 +155,7 @@ public void testFiringEntryModifiedEventWhenModifyingEntryInCache() throws Excep
 
    public void testFiringEntryInvalidatedWhenUsingCacheNotifier() throws Exception {
       //when
-      cache1Notifier.notifyCacheEntryInvalidated("pete", "Edinburgh", true, invocationContext, null);
+      cache1Notifier.notifyCacheEntryInvalidated("pete", "Edinburgh", null, true, invocationContext, null);
```

Comment:
Ok, I was worried this might have been an actual bug in the current implementation.